UIKitDZ/UIKitDZ/Choose Coffee screen/Coffee Beans Screen/CoffeeBeansController.swift

```diff
@@ -0,0 +1,164 @@
+// CoffeeBeansController.swift
+// Copyright © RoadMap. All rights reserved.
+
+import UIKit
+
+//Контроллер с выбором обжарки зерен
+final class CoffeeBeansController: UIViewController {
+    // var delegate: SelectionRoastCoffeDelegate?
+    // var selectedRoastText: String?
+
+    // MARK: - Visual Component
+
+    // кружок наполовину закрытый
+    private lazy var circleWhite: UIImageView = {
+        let imageView = UIImageView(frame: CGRect(x: 24, y: 50, width: 44, height: 44))
+        imageView.image = UIImage(named: "circleWhiteImage")
+        return imageView
+    }()
+
+    // новая вью
+    private lazy var newView: UIView = {
+        let view = UIView(frame: CGRect(x: 0, y: 62, width: 375, height: 750))
+        view.backgroundColor = .white
+        view.layer.cornerRadius = 15
+        return view
+    }()
+
+    // крестик - закрыть
+    private lazy var xButton: UIButton = {
+        let button = UIButton(frame: CGRect(x: 20, y: 26, width: 14, height: 14))
+        button.setImage(UIImage(named: "xImage"), for: .normal)
+        return button
+    }()
+
+    // текст "уточните...
+    private lazy var beansText: UILabel = {
+        let label = UILabel(frame: CGRect(x: 40, y: 53, width: 294, height: 30))
+        label.text = "Уточните обжарку зеренъ"
+        label.font = UIFont(name: "Verdana-Bold", size: 14.0)
+        label.textAlignment = .center
+        return label
+    }()
+
+    // !темная обжарка!
+    // основание
+    private lazy var rectangleDark: UIImageView = {
+        let imageView = UIImageView(frame: CGRect(x: 15, y: 102, width: 165, height: 165))
+        imageView.image = UIImage(named: "rectangleBeans")
+        return imageView
+    }()
+
+    // картинка зерен
+    private lazy var beansDark: UIImageView = {
+        let imageView = UIImageView(frame: CGRect(x: 31, y: 17, width: 100, height: 100))
+        imageView.image = UIImage(named: "darkBeanse")
+        return imageView
+    }()
+
+    // кнопка
+    private lazy var darkRoastsButton: UIButton = {
+        let darkButton = UIButton(type: .custom)
+        darkButton.frame = CGRect(x: 0, y: 112, width: 165, height: 40)
+        darkButton.setTitle("Темная\nобжарка", for: .normal)
+        darkButton.titleLabel?.numberOfLines = 2
+        darkButton.titleLabel?.textAlignment = .center
+        darkButton.setTitleColor(.black, for: .normal)
+        darkButton.titleLabel?.font = UIFont(name: "Verdana", size: 14)
+        // darkButton.addTarget(self, action: #selector(darkRoastButtonTapped), for: .touchUpInside)
+        return darkButton
+    }()
+
+    // !светлая обжарка!
+    // основание
+    private lazy var rectangleLight: UIImageView = {
```

Comment:
```suggestion
    private lazy var rectangleLightImageView: UIImageView = {
```
везде поправь по всем вью